### ISSUE TITLE
ADV190023 | Microsoft upcoming security patch

### ISSUE THREAD

**Issue by nowayi2020** [BeyondTrust/pbis-open#270]
Hi,

I just would like to query some info about the impact on pbis with the upcoming microsoft security change\patch? My main perspective is mainly linux client side.

Is there any info about this?

Regards,
NoWayi

**Comment by rbest-bt** [comment]
We are working without issues. We have enabled this in our testing labs for a few months now.

**Comment by rbest-bt** [comment]
We discovered that this is not going to work with the current release. We have addressed this in the Enterprise release which is going out later this month.

**Comment by nowayi2020** [comment]
Thanks for the info! I presume it will be also addressed in the open version as well?

**Comment by viralmutant** [comment]
> We discovered that this is not going to work with the current release. We have addressed this in the Enterprise release which is going out later this month.

Are we broken with that MS patch ? The prior comment says that we are working without issues. What exactly breaks ?

**Comment by rbest-bt** [comment]
Some lookups are not working. The following will return "ERROR: 500008: Strong(er) authentication required"
> adtool -a search-user --name=*

This has been resolved with recent work on the krb5 update in Enterprise. The method we used will not work with adb-open and we will need to look at a different method for adb-open. Hopefully later this year.

In the meantime, I would suggest enabling the advisory in a test environment and see how it impacts your workflow.
